internal/resources/istio_resources_finder.go

```diff
@@ -89,6 +90,10 @@ func (i *IstioResourcesFinder) FindUserCreatedIstioResources() ([]Resource, erro
 			return nil, err
 		}
 		for _, item := range u.Items {
+			val, exists := item.GetLabels()[moduleLabels.ModuleLabelKey]
```

Comment:
We could also move this logic into a function (e.g. `HasModuleLabel(unstructured.Unstructured)` or `IncludesModuleLabel(map[string]string)` )in the labels package, because it might come handy in the future for other implementation as well.